scylla-cql/src/frame/request/auth_response.rs

```diff
@@ -1,37 +1,22 @@
 use crate::frame::frame_errors::ParseError;
 use bytes::BufMut;
-use std::convert::TryInto;
 
 use crate::frame::request::{Request, RequestOpcode};
-use crate::frame::Authenticator;
 
 // Implements Authenticate Response
 pub struct AuthResponse {
-    pub username: Option<String>,
-    pub password: Option<String>,
-    pub authenticator: Authenticator,
+    pub response: Option<Vec<u8>>,
+    pub authenticator: String,
```

Comment:
The `AUTH_RESPONSE` frame contains only one field. The `authenticator` does not belong here.